superduperdb/container/model.py

```diff
@@ -203,11 +203,11 @@ def _predict_with_select(
     ):
         ids = []
         if overwrite:
-            query = select.select_ids
+            query = select.select_ids()  # TODO: is this right?
```

Comment:
This fix is not correct. (Original is correct.) Was this flagged by `mypy` or similar?